src/bindings/c/include/openvino/c/ov_model.h

```diff
@@ -127,13 +247,13 @@ ov_model_reshape_by_ports(const ov_model_t* model, size_t* ports, const ov_parti
  * @brief Do reshape in model with a list of <ov_output_node_t, partial shape>.
  * @ingroup model
  * @param model A pointer to the ov_model_t.
- * @param output_nodes The ov_output_node_t list.
+ * @param output_ports The ov_output_node_t list.
  * @param partialShape A PartialShape list.
  * @param cnt The item count in the list.
  */
 OPENVINO_C_API(ov_status_e)
 ov_model_reshape_by_nodes(const ov_model_t* model,
-                          const ov_output_node_t* output_nodes[],
+                          const ov_output_node_t* output_ports[],
                           const ov_partial_shape_t* partial_shapes[],
```

Comment:
still, why not `const ov_partial_shape_t** partial_shapes`? You are passing array by value.

Reply:
Here issue is fixed in https://github.com/openvinotoolkit/openvino/pull/12591